community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -621,7 +623,7 @@ private void afterRollback()
     {
         try
         {
-            closeTransaction();
+            closeTransaction( -1 );
```

Comment:
Why is this value `-1`? Does it rely on the implementation of `TransactionIdTracker` that is much higher in the stack?